grpc/src/main/java/com/linecorp/armeria/server/grpc/HttpJsonTranscodingService.java

```diff
@@ -347,16 +345,7 @@ private static Map<String, Field> buildFields(Descriptor desc,
                                 .filter(d -> d.getFullName().equals(field.getMessageType().getFullName()))
                                 .findFirst().orElse(null);
                     if (typeDesc == null) {
```

Comment:
Should we also remove `desc.getNestedTypes().stream()...` block?
It seems we directly assign `typeDesc` from `field.getMessageType()` without `@Nullable`.

Reply:
+1